inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_database_mysql.adoc

```diff
@@ -28,21 +28,40 @@ the application.
 ----
 $ apt-get install mysql-server
 ----
-* make sure your MySQL server is configured for UTF-8. Check the following line is present in `/etc/mysql/mariadb.conf.d/50-server.cnf` (this is specific to Debian 9; on other systems the relevant file
-may be `/etc/mysql/my.cnf`):
+
+[[character-set-config]]
+* make sure your MySQL server is configured for 4 Byte UTF-8 (utf8mb4) to ensure that all unicode characters can be represented (e.g. emojis). In MySQL versions 8 and upwards, this is the default. Check that `character-set-server`, `default-character-set` are set to  `utf8mb4` and `collation-server` is set to `utf8mb4_unicode_ci` by logging into MySQL with
```

Comment:
In newer versions of mysql, it is (utf8mb4, utf8mb4_0900_ai_ci) so I would write either (utf8mb4, utf8mb4_0900_ai_ci) or (utf8mb4, utf8mb4_unicode_ci). Also, it is dangerous to change collation and charset so I would add a warning.

Reply:
How is it dangerous? I would assume only when the database was already running and you would risk corrupting the content. is it also dangerous when setting up for the first time?

Reply:
If you have no data in there then it is not risky

Reply:
Thanks, I made the changes.